requirements.txt

```diff
@@ -1,3 +1,4 @@
 Flask
 ipaddress
 requests
+gunicorn
```

Comment:
I found it very unstable (server freeze after some time of working) without using a proper wsgi server like gunicorn.